pom.xml

```diff
@@ -46,9 +46,9 @@
     <jacoco.version>0.8.8</jacoco.version>
 
     <!-- Beam and linked versions -->
-    <!-- Also update beam version in requirements.txt throughout the repo -->
-    <beam.version>2.56.0</beam.version>
-    <jackson.version>2.14.1</jackson.version>
+    <beam-java.version>2.56.0</beam-java.version>
+    <beam-python.version>${beam-java.version}</beam-python.version>
```

Comment:
imo we can keep `${beam.version}` and create `<beam-python.version>`. This minimizes the footprint of change and avoid any unexpected / missing places need to make change.